tests/bootstrap.php

```diff
@@ -2,6 +2,9 @@
 
 require_once dirname(__DIR__).'/vendor/autoload.php';
 
+// Disable garbage collector to prevent segfaults
+gc_disable();
+
 set_include_path(get_include_path().PATH_SEPARATOR.dirname(__DIR__).'/lib');
```

Comment:
`PATH_SEPARATOR` is not `DIRECTORY_SEPARATOR`. `PATH_SEPARATOR` contains `:` on Linux and Mac OS.